slides/pages/signup.ftd

```diff
@@ -0,0 +1,103 @@
+-- import: ui.fifthtry.com/slides/components/action-page
```

Comment:
@MeenuKumari28 you have duplicated entire code login.ftd. Also rename `login.ftd` to `signin.ftd`, we can have `login` and `signup`. `login` goes with `register`, and `signin` goes with `signup`.